### ISSUE TITLE
Bang not work with Firefox

### ISSUE THREAD

**Issue by bepolymathe** [QwantResearch/qwantmaps#89]
I am an intensive user of the "bangs" in my browser. "g + summer" searches for the term "summer" on google for example. 

I wanted to do the same on qwant maps but it doesn't work. To reproduce. 

1. Open firefox
2. Go to https://www.qwant.com/maps
3. Click right on the search field and "add a key word for search"

![image](https://user-images.githubusercontent.com/29939805/61931744-d3c42580-af81-11e9-92f5-de7e1611cb4f.png)

4. Type "m" for map in key word and save
5. Open a new tab and type "m paris" + enter

In this case, I have the word "paris" added in the address bar but the search is not done. 

`https://www.qwant.com/maps/?=paris`

![image](https://user-images.githubusercontent.com/29939805/61932165-f0ad2880-af82-11e9-9abd-de10e952e770.png)

Could you tell us what the search address in qwant maps is so we can make functional bangs?

Thank you in advance.

**Comment by bepolymathe** [comment]
Well, I figured it out on my own. I put it here for documentation and for it to be integrated by default. This is probably a problem with the search field dialog box.

A "q" must be added after the "?" in the search address.

![image](https://user-images.githubusercontent.com/29939805/61932634-2868a000-af84-11e9-9d12-30682cd0c54d.png)

**Comment by nlehuby** [comment]
I didn't know about this "add keyword for this search" feature in Firefox!

The opensearch standard is implemented for Qwant Maps: in Firefox, you can click on the search bar to add Qwant Maps as a search engine, and it will use the correct syntax (with the q at the end).
You can then select a keyword for the search in `about:preferences`.
![qmaps](https://user-images.githubusercontent.com/919962/61933984-86e34d80-af87-11e9-9bb8-0b9339668770.png)

**Comment by bepolymathe** [comment]
Yes, the flying Dutchman (le hollandais volant) speaks very well about it (in French). 
https://lehollandaisvolant.net/?d=2017/10/03/17/55/17-faire-des-bang-dans-firefox
